pipeline_tools/get_analysis_workflow_metadata.py

```diff
@@ -25,34 +24,24 @@ def get_analysis_workflow_id(analysis_output_path):
     return workflow_id
 
 
-def get_auth(credentials_file=None):
-    """Parse cromwell username and password from credentials file.
-
-    Args:
-        credentials_file (str): Path to the file containing cromwell authentication credentials.
-
-    Returns:
-        requests.auth.HTTPBasicAuth: An object to be used for cromwell requests.
-    """
-    credentials_file = credentials_file or '/cromwell-metadata/cromwell_credentials.txt'
-    with open(credentials_file) as f:
-        credentials = f.read().split()
-    user = credentials[0]
-    password = credentials[1]
-    return HTTPBasicAuth(user, password)
+def get_headers():
+    credentials, project = google.auth.default()
+    if not credentials.valid:
+        credentials.refresh(google.auth.transport.requests.Request())
+    headers = {}
+    credentials.apply(headers)
+    return headers
 
 
-def get_metadata(cromwell_url, workflow_id, http_requests, use_caas=False, caas_key_file=None):
-    """Get metadata for analysis workflow from Cromwell and write it to a JSON file. Retry the request with
+def get_metadata(cromwell_url, workflow_id, http_requests):
```

Comment:
nit: It would be great to add `this function only works with OAuth-enabled Cromwell Instances` to the docstring.